en/tutorials/news-1-getting-started.md

```diff
@@ -33,24 +33,33 @@ In this part we will start with a minimal Vespa application to
 get used to some basic operations for running the application on Docker.
 In the next part of the tutorial, we'll start developing our application.
 
-
-
 ## Prerequisites
 
-- [Docker Desktop on Mac](https://docs.docker.com/docker-for-mac/install) 
-  or Docker on Linux
-- [Git](https://git-scm.com/downloads) - or download the
-  [sample apps](https://github.com/vespa-engine/sample-apps/archive/master.zip)
-- python 3
-- Operating system: macOS or Linux
-- Architecture: x86_64
-- Minimum **10 GB** memory dedicated to Docker (the default is 2 GB on Macs)
+* [Docker](https://www.docker.com/) Desktop installed and running. 6GB available memory for Docker is recommended.
+  Refer to [Docker memory](https://docs.vespa.ai/en/operations/docker-containers.html#memory)
+  for details and troubleshooting
+* Operating system: Linux, macOS or Windows 10 Pro (Docker requirement)
+* Architecture: x86_64
+* Minimum **10 GB** memory dedicated to Docker (the default is 2 GB on Macs)
```

Comment:
says 6G above, probably remove the 6G as I seem to remember 10G was required for this tutorial?